Cargo.toml

```diff
@@ -11,25 +11,29 @@ readme = "README.md"
 categories = ["gui", "rendering"]
 
 [dependencies]
-winapi = { version = "0.3", features = [
-    "d3d11",
-    "dxgitype",
-    "dxgiformat",
-    "dxgi",
-    "minwindef",
-    "winerror",
+windows = { version = "0.36", features = [
+    "Win32_Graphics_Direct3D11",
+    "Win32_Graphics_Direct3D",
+    "Win32_Graphics_Dxgi",
+    "Win32_Graphics_Dxgi_Common",
+    "Win32_Foundation"
 ] }
-wio = "0.2"
-imgui = "0.7"
+imgui = "0.8"
 
 [build-dependencies]
 winapi = { version = "0.3", features = ["d3dcompiler", "d3dcommon"] }
```

Comment:
I see you haven't ported the build script to `windows`, is that because the required parts aren't available? Or just because you didn't have the need to do that? (it's fine either way, just asking out of curiosity)

Reply:
Laziness mostly, I'll do that.

Reply:
Done